//装备熔炉消息
import "MDropList.proto";
message DseReforgeEquip
{
	optional int32		action 	= 1;//操作类型，1=还原装备，2=分解装备
	repeated string		tid 	= 2;//装备唯一ID
	optional MDropList	list	= 3;//返还的物品列表
	optional int32		code	= 4;//错误码
}